### ISSUE TITLE
Language support

### ISSUE THREAD

**Issue by larsenwork** [andreaslarsen/animalnoteheads#6]
I'd like to support as many languages as possible. The ones with checkmarks are done. 

Please add your suggestions in a comment below and note that : 
- Use as many of the existing icons as possible for your list
- Think about colors and contrast  
  - Some animals (e.g. cat, dog) can have different colours and still look good whereas others (e.g. elefant) only works in one color.  
  - Not that important if the animals are far apart (e.g. yellow Cat and orange Giraf)
- Avoid subspecies as kids will often guess wrong
  - I had to use Chimpanzee in Danish (only real alternative was chinchilla) - but many kids will guess "monkey"  
- The head of the animal should have a distinct look (e.g. dolphin is very difficult to draw because kids are used to see them drawn from the side and typically with the entire body)

<div id="current"></div>
## Current icons

(animal alphabet): alligator, ant, baboon, bat, bear, bee, cat, chimpanzee, dear, dog, dragon, duck, elephant, fish, giraffe, lion, lizard, monkey, owl, raccoon, seal, snake, dinosaur
## Languages
- [x] English - Alligator, Bear, Cat, Dog, Elephant, Fish, Giraffe
- [x] Français - Lion, Souris, Dragon, Raton, Mouton, Fourmi, Serpent
- [x] Español - León, Sapo, Dragón, Ratón, Mapache, Foca, Serpiente
- [x] Ελληνικά - Λιοντάρι, Σαύρα, Νυχτερίδα, Ρακούν, Μέλισσα, Φώκια, Σκύλος
- [x] Dansk - And, Hund, Chimpanse, Drage, Elefant, Fisk, Giraf 
- [x] Svenska - Anka, Hundar, Chinchilla, Drake, Elefant, Fisk, Giraff
- [x] Norsk - And, Hund, Chinchilla, Drage, Elefant, Fisk, Giraf
- [x] Deutsch - Affe, Hund, Chamäleon, Drache, Elefant, Fisch, Giraffe
- [x] Nederlands - Aap, Bij, Chimpansee, Dinosaurus, Eend, Flamingo, Giraf
- [ ] Polski
- [ ] Русский
- [ ] Italiano
- [ ] Portuguese
- [ ] Română
- [ ] Hrvatski 
  etc...

**Comment by elijahverdoorn** [comment]
Spanish:
- A: abeja (bee)
- B: búho (owl)
- C: ciervo (deer)
- D: delfín (dolphin)
- E: elefante (elephant)
- F: foca (seal)
- G: gato (cat)

**Comment by yroul** [comment]
- French
  - A: Alligator (Alligator)
  - B: Baleine (Whale)
  - C: Chat (Cat)
  - D: Dauphin (Dolphin)
  - E: Éléphant (Elephant)
  - F: Fourmi (Ant)
  - G: Girafe (Giraf)

**Comment by larsenwork** [comment]
@elijahverdoorn thx, they should be doable except maybe the dolphin - any alternate suggestions for D? Dragon?

**Comment by larsenwork** [comment]
@yroul thank you - could you think of a couple of alternatives for B and D ?

**Comment by yroul** [comment]
You'r welcome ! The first alternative that come to my mind for D and B are
"Dragon" and "Babouin" (it's a little monkey = baboon )
Let me know if you think these two still not doable, i'll search for others.

Regards,
Yoan

**Comment by larsenwork** [comment]
@yroul 

I thought about changing the color of the dragon so there aren't two green animals (maybe purple) but other than that I it looks ok. Any thoughts?
![screen shot 2015-05-16 at 18 28 28](https://cloud.githubusercontent.com/assets/3859425/7666962/6d8fe1f4-fbf9-11e4-973b-9307be1c81ff.png)

**Comment by yroul** [comment]
Looks great, and i like the idea of changing dragon's color to avoid confusion.
A red or purple dragon should looks cool.

**Comment by andreaslarsen** [comment]
![screen shot 2015-05-16 at 18 55 18](https://cloud.githubusercontent.com/assets/3859425/7667102/275a0c6a-fbfd-11e4-8960-04b1a6089f8e.png)

It was too much with the flames together with the purple but I think it works quite well without, or?

**Comment by larsenwork** [comment]
![screen shot 2015-05-16 at 18 55 18](https://cloud.githubusercontent.com/assets/3859425/7667102/275a0c6a-fbfd-11e4-8960-04b1a6089f8e.png)

@yroul 
Thx - I'm also quite satisfied with how they turned out. 
It was too much with the flames together with the purple but I think it works quite well without, or?

**Comment by yroul** [comment]
Yes it's works without the flames : i asked my wife and she tell me "Dragon" without doubt.

**Comment by larsenwork** [comment]
@yroul fantastic...that would've been the next thing I would've asked you to do (to show someone the animals without telling them what they were) so french is now done :+1: 

Thank very much for your help! What's your full name so I can add you to the list of translators on the [website](http://animalnoteheads.com/#translators) (include link to your twitter/website if you want)

I'll send out the iOS app to beta testers on monday if you're interested in that #8

**Comment by larsenwork** [comment]
@elijahverdoorn I have finished drawing the spanish animals (the owl still needs some work) 
The deer and the owl had too similar colors so I used crocodile instead of deer. Let me know if you want a link to one of your online profiles when I add your name to the list of translators.

![screen shot 2015-05-16 at 21 23 09](https://cloud.githubusercontent.com/assets/3859425/7667679/d5114472-fc11-11e4-89ad-94c28d3289eb.png)

**Comment by elijahverdoorn** [comment]
@andreaslarsen Looks good! I'd love a link to my twitter: https://twitter.com/elijahverdoorn

Anything else I can do on this project? I'm a programmer too, where do you need help?

**Comment by larsenwork** [comment]
@elijahverdoorn that sounds great, there are many things you can help with depending on which languages you know/want to play around with.

Pianimo (iOS app) - coded in swift - I'll upload it as the next thing (it has it's own repo)

Website/webapp for browsing + viewing the sheets (I'll create a new issue for this)

Build scripts - when abraham has the core lilypond functions working (should be very soon) then we'll probably need to create some extra scripts to automate as much of the process as possible.

The owl ended up look like this by the way: 
![screen shot 2015-05-17 at 01 38 26](https://cloud.githubusercontent.com/assets/3859425/7668376/747ce7aa-fc35-11e4-9f73-6100568ca815.png)

**Comment by yroul** [comment]
@andreaslarsen You're welcome !
I'd like a link to my website http://yoanroullard.fr/.
Let me know if i can help on something else : i'm a musician and programmer too.
My skills are mostly HTML5/CSS3/JS programming, but maybe i can help on others topics.

**Comment by larsenwork** [comment]
@yroul I already added link to your website - found it on your github profile :wink: 

It great that you want to help - you can see all issues here https://github.com/andreaslarsen/animalnoteheads/issues

I think #10 is where your skills are best used

**Comment by larsenwork** [comment]
Ντο(C)-Νυχτερίδα/bat
Ρε (D)-Ρινόκερος/rhino
Μι(E)-Μέλισσα/bee
Φα(F)-Φώκια/seal
Σολ(G)-Σκύλος/dog
Λα (A)-Λιοντάρι-lion
Σι(B)-Σαύρα-lizzard(you could use an alternate dragon sprite)

**Comment by larsenwork** [comment]
![screen shot 2015-05-17 at 18 09 20](https://cloud.githubusercontent.com/assets/3859425/7671129/e92ede3e-fcbf-11e4-9322-bff8f8ae6bc7.png)

I need another greek animal starting with Ρ - the bat + rhino are too similar in color

**Comment by larsenwork** [comment]
![master](https://cloud.githubusercontent.com/assets/3859425/7679151/859ee08a-fd5a-11e4-8bed-76f97fa502b4.png)

Does anyone know an alternative to racoon + rhino for greek ?

**Comment by larsenwork** [comment]
@iccir see the update bear + giraffe

**Comment by yroul** [comment]
@andreaslarsen i just had a moment of clarity : 

French note are not called A B C D E F G but La Si Do Ré Mi Fa Sol.

I completely forget to tell you because i mostly use the english names.

Does it matter ?

**Comment by larsenwork** [comment]
@yroul hehe...it depends on what names you generally use in france when learning them?

**Comment by yroul** [comment]
Definitly "Do-Ré-Mi ..."

It's almost impossible to find easy animal name (= known by kids) which match perfectly the note name.
Maybe we can just use the first letter : 

Do : Dauphin (Dolphin) or Dragon
Ré : Rat
Mi : Macaque (can reuse the baboon design) or Marmotte (Marmot)
Fa : Fourmi (Ant)
Sol : Serpent(Snake)
La : Lapin(Rabbit) or Lion
Si : Singe (Monkey - if we not use "Macaque" for Mi) or Souris (Mouse)

"Sol" and "Si" start with the same letter...it can be confusing

Update to avoid confusion : 

C = Do
D = Ré
E = Mi
F = Fa
G = Sol
A = La
B = Si

**Comment by larsenwork** [comment]
@yroul I have already drawn a mandrill we can use and rat and mouse are too similar so maybe

Dragon
Rat
Mandrill
Fourmi
Serpent
Lion 
Singe

**Comment by larsenwork** [comment]
@elijahverdoorn do you also use "do re mi ..." when learning notes in Spain?

**Comment by elijahverdoorn** [comment]
@andreaslarsen I actually don't know. I'm from the United States, I just happen to be studying Spanish in University to fulfill my language requirement. I can ask my Spanish professor if you like.

**Comment by larsenwork** [comment]
that'd be great :)

**Comment by larsenwork** [comment]
@yroul can you think of another animal with R?

Problem is I want the animals to look happy and rats are drawn just like mouse - only looking angry/evil :)

**Comment by yroul** [comment]
@andreaslarsen "Raton Laveur" <=> Raccoon. They're cute :).

@andreaslarsen @elijahverdoorn Be careful, according these pages http://en.wikipedia.org/wiki/Musical_note
http://es.wikipedia.org/wiki/Nota_(sonido)

Spanish Language use the same system as we do in France

**Comment by larsenwork** [comment]
@yroul yeah, I suspected it was the same for all south-western european languages :) 

I just drew a racoon for greek so that was easy ;)

**Comment by elijahverdoorn** [comment]
Sounds good. My professor agrees, she was taught in Argentina and they learned with this system. Here is a revised list:
Do: delfin (dolphin)
Re: Raton (mouse)
MI: Mariposa (butterfly)
Fa: Focal (seal)
Sol: sapo (toad/frog)
La: Lagarto (lizard)
Si: Serpent (snake)

I went with the first letter idea also as finding easy animals was challenging for me too. Let me know if you want alternates for any of these.

**Comment by larsenwork** [comment]
dragon 
raton
mapache (raccoon?)
focal

frog, lizard and snake share the same characteristics so if possible only one of them and something else for the other two :)

**Comment by elijahverdoorn** [comment]
@andreaslarsen Let's try this:

Do: dragón (dragon)
Re: Raton (mouse)
MI: mapache (raccoon)
Fa: Focal (seal)
Sol: sapo (toad/frog)
La: lobo (wolf)
Si: Serpent (snake)

Not sure about the Sol/Si. "s" is not that common in spanish animals, I'll ask around.

**Comment by larsenwork** [comment]
ok, I'll maybe use léon for the L since I already have that :smile:

**Comment by elijahverdoorn** [comment]
Fine with me. :+1:

**Comment by larsenwork** [comment]
![master](https://cloud.githubusercontent.com/assets/3859425/7702732/abfb32aa-fe32-11e4-8c68-a482a7af3e6f.png)

Spanish + french update

**Comment by yroul** [comment]
My thoughts : 
- Is it possible to add a mane to the lion ? It looks more like a cat to me.
- For the mandrill, the kids may say "Monkey" rather than mandrill  until someone explain "No, this is a mandrill, like Rafiki in The Lion King)

**Comment by larsenwork** [comment]
ah, that's because it's the monkey image on not mandrill - my mistake :) 

lion: yeah...it's one of the icons I'm trying to improve

**Comment by larsenwork** [comment]
![master](https://cloud.githubusercontent.com/assets/3859425/7704164/11446f46-fe3d-11e4-80d4-4cafebbf55b2.png)

**Comment by larsenwork** [comment]
I also tried with a mane but it had to be too big to really work so went with a "nala" from lion king look instead

**Comment by yroul** [comment]
Better :)

**Comment by larsenwork** [comment]
yep, only thing about the mandrill is it's a bit "busy" to look at because it needs so many colors...maybe a bit better without the dark around the eyes...
![screen shot 2015-05-19 at 15 50 51](https://cloud.githubusercontent.com/assets/3859425/7704470/da10db84-fe3e-11e4-9b24-0eb748bc8c56.png)

do you have other common animals that begin with m?

**Comment by yroul** [comment]
Marmotte = Marmot
Mouton = Sheep
Mouche = Fly

**Comment by iccir** [comment]
@andreaslarsen - The bear's lighter tone is definitely better! :)

**Comment by larsenwork** [comment]
@iccir thx :) 

Le mouton
![screen shot 2015-05-20 at 12 48 24](https://cloud.githubusercontent.com/assets/3859425/7724101/bf5d53a2-feee-11e4-822e-20a5e5ddccfb.png)

**Comment by yroul** [comment]
Perfect :)

**Comment by larsenwork** [comment]
the iOS app is now also localised - send me your email if you want to try :) 

![ios simulator screen shot 20 may 2015 15 16 33](https://cloud.githubusercontent.com/assets/3859425/7726560/52f75432-ff03-11e4-886e-41e6767be96c.png)

**Comment by larsenwork** [comment]
Can @elijahverdoorn and @yroul translate this? It's for the iOS app for the free Pianimo app (screenshot above)

> Pianimo teaches kids of all ages the names of the notes on the piano. Alligator for A, Bear for B, etc. The tricycle and bicycle helps visualise the groups of 3 and 2 black keys.
> 
> It changes animal icons based on your iPhone/iPad language settings. 
> 
> AnimalNoteHeads.com is an open source project where your ideas and skills are more than welcome! We'll soon add some sheets using the AnimalNoteHeads too.
> 
> Supported languages:
> ...
> Is your language missing? Help me on animalnoteheads.com/language

Edit: Forgot the keywords

> Piano, kids, notes, animals

**Comment by yroul** [comment]
_Pianimo permet d'apprendre aux enfants de tous âges le nom des notes sur le piano. Dragon pour le Do, Raton pour le Ré etc... Le tricycle et le vélo aident à visualiser les groupes de 3 et 2 touches noires._

_Les animaux changent en fonctions de la langue choisie dans les préférences de votre iPhone/iPad._

_AnimalNoteHeads.com est un projet open source pour lequel vos idées et vos compétences sont plus que les bienvenues. Nous ajouterons bientôt des partitions utilisant les "AnimalNoteHeads"._

*Langues supportées : *
....

_Piano, enfants, notes, animaux_

I've made the choice to not translate "AnimalNoteHeads" in the sentence "We'll soon add some sheets using the AnimalNoteHeads too." because it's the name of the project, but i can do it if needed.

**Comment by larsenwork** [comment]
Perfect!!

**Comment by larsenwork** [comment]
@yroul and @elijahverdoorn as far as I could deduct from the wikipedia pages G-clef also uses Sol, right? 

Just discovered that I hadn't shifted the notes two times to the right for the latin languages so Do = A instead of C...fix coming very soon :hurtrealbad:

**Comment by elijahverdoorn** [comment]
Here is your translation:

Pianimo enseña a los niños (todos los edades) los nombres de las notas de la música. Cocodrilo por A, oso para B, etc. El triciclo y bicicleta ayuda a visualizar los grupos de 3 y 2 teclas negras.

Cambia iconos de animales sobre la configuración de idioma en tu iPhone / iPad.

AnimalNoteHeads.com es un proyecto de código abierto en el que sus ideas y habilidades son bienvenidos! Pronto añadiremos algunas hojas utilizando las AnimalNoteHeads también.

Idiomas soportados:
...
¿No veo tu idioma? Ayuda en animalnoteheads.com/language

**Comment by larsenwork** [comment]
:+1:

**Comment by larsenwork** [comment]
Added german, norwegian and swedish
![animalnoteheads](https://cloud.githubusercontent.com/assets/3859425/8397484/e1b89afe-1dcc-11e5-911c-011c37700377.png)

**Comment by larsenwork** [comment]
Polish names:
A - Antylopa | Aligator | Anakonda
B - Baran | Byk | Bizon | Borsuk | Bóbr | Błotniarka
C - Chrząszcz | Czapla | Czajka | Cielak | Czerwoniak
D - Dinozaur | Dudek | Delfin 
E - Emu
F - Flaming | Foka | Foksterier 
G - Gołąb | Goryl | Gęś | Gil | Gąsienica | Gepard | Gazela
H - Hipopotam | Hiena | Homar | Hoacyn | Harpia

**Comment by larsenwork** [comment]
Polish in englsh
A - Antelope | Alligator | Anaconda
B - Aries | Taurus | Bison | Badger | Beaver | Błotniarka
C - Beetle | heron | Chaika | calf | Czerwoniak
D - Dinosaur | Hoopoe | Dolphin
E - Emu
F - Flaming | Seal | Fox Terrier
G - Dove | gorilla | Goose | Gil | caterpillar | Cheetah | Gazelle
H - Hippopotamus | hyena | lobster | Hoatzin | Harpy
